src/config/config.ts

```diff
@@ -333,15 +334,17 @@ export class Config implements IConfig {
 
     this.npmRegistry = this.scopedEnvVar('NPM_REGISTRY') || this.pjson.oclif.npmRegistry
 
-    if (!this.scopedEnvVarTrue('DISABLE_THEME')) {
-      const {theme} = await this.loadThemes()
-      this.theme = theme
+    this.theme = await this.loadTheme()
+
+    const s3 = this.pjson.oclif.update?.s3 ?? {
```

Comment:
I don't like this style (declaring a const object with placeholders).

could there be a function whose single responsibility is to build the `s3` object, or the `updateConfig` object?

Reply:
and if the property is optional, we could type it that way rather than using empty string